tensorboard/plugins/hparams/api_test.py

```diff
@@ -240,5 +244,111 @@ def test_dtype_mismatch(self):
       hp.Discrete(["one", 2])
 
 
+class KerasCallbackTest(test.TestCase):
+  def setUp(self):
+    super(KerasCallbackTest, self).setUp()
+    HP_DENSE_NEURONS = hp.HParam("dense_neurons", hp.IntInterval(4, 16))
+    self.hparams = {
+        "optimizer": "adam",
+        HP_DENSE_NEURONS: 8,
+    }
+    self.model = tf.keras.models.Sequential([
+        tf.keras.layers.Dense(self.hparams[HP_DENSE_NEURONS], input_shape=(1,)),
+        tf.keras.layers.Dense(1, activation="sigmoid"),
+    ])
+    self.model.compile(loss="mse", optimizer=self.hparams["optimizer"])
+    self.logdir = os.path.join(self.get_temp_dir(), "logs")
+    self.callback = hp.KerasCallback(
+        self.logdir,
+        self.hparams,
+        group_name="psl27",
+    )
+
+  @test_util.run_v2_only("Requires eager mode.")
```

Comment:
You can still run this in TF 1.0 if we just set `tf.compat.v1.enable_eager_execution()` at the top of the test case?  And then you can use `with tf.compat.v1.Graph().as_default()` to get back into graph mode for the non-eager test case.

This is what the summary tests do, see https://github.com/tensorflow/tensorboard/blob/992bd022ed95c994d2f97f206a3c27bd5e351763/tensorboard/plugins/scalar/summary_test.py#L42